src/jit/compiler.h

```diff
@@ -6931,6 +6929,17 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     // Should we support SIMD intrinsics?
     bool featureSIMD;
 
+    // Have we identified any SIMD types?
+    bool _usesSIMDTypes;
+    bool usesSIMDTypes()
```

Comment:
Should this be a PhasedVar? Is it legal to read the value and then change it later?